### PR TITLE
Fix bug  in topic modal dialog and support facebook and project_github

### PR DESCRIPTION
**Pull request by railsstudent**
1) blog link can click and open blog in current tab
2) Add facebook icon for Japanese speaker
3) Add project_github. (Sorry, something I invented)

### CODE REVIEW

File: assets/js/timetable/component/ModalProvider.jsx [hunk 1]
```diff
@@ -71,16 +71,26 @@ export default class ModalProvider extends React.Component {
...
+                  </a>
+                )}
                 {hasSocial && 'twitter' in speaker.social && (
                   <a href={speaker.social.twitter} className="btn-floating blue-theme">
                     <i className="fa fa-twitter" />
                   </a>
                 )}
                 {hasSocial && 'blog' in speaker.social && (
-                  <a href={speaker.blog} className="btn-floating blue-theme">
+                  <a href={speaker.social.blog} className="btn-floating blue-theme">
```
Comment (tonyhhyip) [inline assets/js/timetable/component/ModalProvider.jsx]: I think open new tab is better

File: assets/js/timetable/component/ModalProvider.jsx [hunk 2]
```diff
@@ -71,16 +71,26 @@ export default class ModalProvider extends React.Component {
...
                     <i className="fa fa-globe" />
                   </a>
                 )}
+                {hasSocial && 'facebook' in speaker.social && (
+                  <a href={speaker.social.facebook} className="btn-floating blue-theme">
```
Comment (tonyhhyip) [inline assets/js/timetable/component/ModalProvider.jsx]: I think open new tab is better

File: assets/js/timetable/component/ModalProvider.jsx [hunk 3]
```diff
@@ -71,16 +71,26 @@ export default class ModalProvider extends React.Component {
                     <i className="fa fa-github globe" />
                   </a>
                 )}
+                {hasSocial && 'project_github' in speaker.social && (
+                  <a className="btn-floating blue-theme" href={`${speaker.social.project_github}`} target="_blank">
+                    <i className="fa fa-github globe" />
+                  </a>
+                )}
                 {hasSocial && 'twitter' in speaker.social && (
                   <a href={speaker.social.twitter} className="btn-floating blue-theme">
```
Comment (tonyhhyip) [inline assets/js/timetable/component/ModalProvider.jsx]: I think open new tab is better

**Review by tonyhhyip — CHANGES_REQUESTED** [review]

**Comment by railsstudent** [comment]
Fixed.

**Review by tonyhhyip — APPROVED** [review]

**Pull request merged**